parquet-column/src/test/java/parquet/schema/TestTypeBuilders.java

```diff
@@ -599,6 +601,479 @@ public Type call() throws Exception {
         });
   }
 
+  @Test
+  public void testRequiredMap() {
+    List<Type> typeList = new ArrayList<Type>();
+    typeList.add(new PrimitiveType(REQUIRED, INT64, "key"));
+    typeList.add(new PrimitiveType(REQUIRED, INT64, "value"));
+    GroupType expected = new GroupType(REQUIRED, "myMap", OriginalType.MAP, new GroupType(REPEATED,
+        "map",
+        typeList));
+    GroupType actual = Types.requiredMap()
+        .key(INT64)
+        .requiredValue(INT64)
+        .named("myMap");
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void testOptionalMap() {
+    List<Type> typeList = new ArrayList<Type>();
+    typeList.add(new PrimitiveType(REQUIRED, INT64, "key"));
+    typeList.add(new PrimitiveType(REQUIRED, INT64, "value"));
+    GroupType expected = new GroupType(OPTIONAL, "myMap", OriginalType.MAP, new GroupType(REPEATED,
+        "map",
+        typeList));
+    GroupType actual = Types.optionalMap()
+        .key(INT64)
+        .requiredValue(INT64)
+        .named("myMap");
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void testMapWithInt64KeyAnnotations() {
```

Comment:
This doesn't appear to have any annotations on the INT64 key field. It looks like this might be misnamed because it looks like it's testing the group builder's `requiredMap` method.